### PR TITLE
Service/news

### PR DESCRIPTION
**Pull request by molleer**

### CODE REVIEW

File: api/services/news/README.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# News
+
+Manages news on chalmers.it
+
+## Development
+
+```
+  docker-compose up news news-db
```
Comment (Portals) [inline api/services/news/README.md]: If the news image `depends_on` news-db, the you only need to specify news.

File: api/services/news/src/models/Cat.ts [hunk 2]
```diff
@@ -0,0 +1,6 @@
+export interface Cat {
```
Comment (Portals) [inline api/services/news/src/models/Cat.ts:1]: 🐈?

File: api/services/news/src/models/Cat.ts [hunk 3]
```diff
@@ -0,0 +1,6 @@
...
+    id: string;
+    name: string;
+    image_url: string;
```
Comment (Portals) [inline api/services/news/src/models/Cat.ts:4]: Only one image?

File: docker-compose.yml [hunk 4]
```diff
@@ -153,4 +153,36 @@ services:
...
+      - news
+    environment:
+      PORT: 5000
+      DB_USER: news
+      DB_NAME: news
+      DB_PASS: password
+      DB_HOST: news-db
+      DB_PORT: 5432
+    ports:
+      - 5000:5000
```
Comment (Portals) [inline docker-compose.yml:177]: Doesn't have to be exposed if used via router

**Review by Portals — COMMENTED** [review]
Great start! Next step maybe is to add some routes in the router?

File: api/services/news/src/models/Cat.ts (hunk 2 above)
Comment (molleer) [reply api/services/news/src/models/Cat.ts:1]: Just a placeholder

File: docker-compose.yml (hunk 4 above)
Comment (molleer) [reply docker-compose.yml:177]: Yes, but is it easier to debug if you can use firefox or insomnia

File: api/services/news/README.md (hunk 1 above)
Comment (molleer) [reply api/services/news/README.md]: That's neat

File: api/services/news/src/models/Cat.ts (hunk 3 above)
Comment (molleer) [reply api/services/news/src/models/Cat.ts:4]: Maybe there should be multiple images for each cat?

File: docker-compose.yml (hunk 4 above)
Comment (Portals) [reply docker-compose.yml:177]: Yeah, I guessed that, but shouldn't be merged into master though.

**Pull request closed without merging**